tests/compilation_tests/database_test.py

```diff
@@ -12,52 +12,59 @@
 
 class TestDataBase(unittest.TestCase):
     # DataBase instances to test on
-    robot_default = Robot(x_pos=0, y_pos=0, heading=0, epsilon=0, max_v=0,
+    robot_state_default = Robot_State(x_pos=0, y_pos=0, heading=0, epsilon=0, max_velocity=0,
                           radius=0)
+    robot_default = Robot(robot_state=robot_state_default)
 
     db_default = DataBase(robot_default)
 
-    robot_initial = Robot(x_pos=0, y_pos=0, heading=0, epsilon=0, max_v=0,
-                          radius=0, plastic_weight=2, move_dist=.6, position_noise=0.23)
-    robot_initial.position_kp = 0.12
-    robot_initial.position_ki = 1.7
-    robot_initial.position_kd = 9.2
-    robot_initial.heading_kp = 0.2
-    robot_initial.heading_ki = 0.4
-    robot_initial.heading_kd = 0.16
-    robot_initial.phase = Phase.TRAVERSE
-    robot_initial.state = np.array([[10], [20], [50]])
-    robot_initial.battery = 98
-    robot_initial.magnetic_field = [0.1, 0.2, 0.3]
-    robot_initial.gyro_rotation = [0.5, 0.2, 0.6]
-    robot_initial.acceleration = [4.25, 3.2, 0.1]
-    robot_initial.is_sim = False
+    # We can't make is_sim false because it fails GitHub merge tests.
+    robot_state_initial = Robot_State(x_pos=0, y_pos=0, heading=0, epsilon=0, max_velocity=0,
+                          radius=0)
+    robot_state_initial.plastic_level = 2
+    robot_state_initial.move_dist = 0.6
+    robot_state_initial.position_noise = 0.23
+    robot_state_initial.position_kp = 0.12
+    robot_state_initial.position_ki = 1.7
+    robot_state_initial.position_kd = 9.2
+    robot_state_initial.heading_kp = 0.2
+    robot_state_initial.heading_ki = 0.4
+    robot_state_initial.heading_kd = 0.16
```

Comment:
yeah... this is clunky - let's change it

Reply:
Maybe we can use **kwargs for the class init so we don't have to assign everything using the dot syntax

Reply:
Good idea!

Reply:
@juliehelenek @AdityaKompella6 what do ug think about **args also for initializing the attributes for our robot in `Robot_State`? Im not too familiar w how **kwargs and **args work so wanna see what ug think